### DISCUSSION TITLE
hoomd installation with both mpi and gpu enabled

### DISCUSSION THREAD

**Discussion by yliu6677** [glotzerlab/hoomd-blue#1828] [Support]
Hello I installed hoomd 4.7 with both mpi and gpu enabled. But when I type `python` in terminal and then `import hoomd`, it says 

```
[yliu-XPS-8960:324458] shmem: mmap: an error occurred while determining whether or not /tmp/ompi.yliu-XPS-8960.1000/jf.0/2002780160/shared_mem_cuda_pool.yliu-XPS-8960 could be created.
[yliu-XPS-8960:324458] create_and_attach: unable to create shared memory BTL coordinating structure :: size 134217728 
``` 

This issue can be avoided by running `import hoomd` in jupyter notebook and create the file manually though... The number in the file is different if I type import hoomd in jupyter notebook or in terminal. So it is just a temporary relief of the problem if I stay in the same jupyter notebook and continue everything. Then it's not a solution to the problem...

**Comment by joaander** [comment]
This is a problem with MPI, not HOOMD. Search for "unable to create shared memory BTL coordinating structure" to find numerous discussions about this problem.

**Reply by yliu6677** [reply]
there are a lot of discussions on this and I didnt really understand how to solve that or if this is an issue that will affect the running somehow... Since I can successfully run tutorials Ill just leave this issue in future when it becomes critical.
